README.md

```diff
@@ -130,3 +130,25 @@ Check `sideEffect/saga.js` for a detail of the inputs and outputs
 * Add unit tests
 * Add `<ReferenceMany>` field
 * Add complex field & input types
+
+## Contributing
+
+### Testing Your Changes on a Real Project
+
+In order to test your changes on a real world project, you can embed a work-in-progress version of the lib in another repository, using `npm-link`:
```

Comment:
~~That's a typo.~~

It is for reffering to official documentation (thanks @fzaninotto :p).